node/src/neighborhood/mod.rs

```diff
@@ -578,9 +584,9 @@ impl Neighborhood {
             &node_descriptor.encryption_public_key,
             node_addr,
         );
-        trace!(
+        debug!(
             self.logger,
-            "Sent Gossip: {}",
+            "Sent Debut Gossip: {}",
             debut_gossip.to_dot_graph(
```

Comment:
These dot graphs should be confined to TRACE logs. They can be pretty big (although I admit not in the Debut case) and shouldn't clutter up DEBUG-level logfiles.